src/wkok/openai_clojure/azure/api.clj

```diff
@@ -0,0 +1,38 @@
+(ns wkok.openai-clojure.azure.api
+  (:require [wkok.openai-clojure.azure.core :as core]))
+
+
+(defn create-completion
+  "Creates a completion for the provided prompt and parameters
+
+  Example:
+  ```
+
+  (create-completion {:model \"text-davinci-003\"
```

Comment:
Just ensure the example works as is, i.e. has all the mandatory params at least, like the `:deployment-id` perhaps?